test/e2e/proxy.go

```diff
@@ -52,6 +52,7 @@ func proxyContext(version string) {
 	f := NewFramework("proxy")
 	prefix := "/api/" + version
 
+	// Port here has to be kept in sync with default kubelet port.
```

Comment:
Kind of a nit/suggestion. What I mean is the next line could be:
```go
It("should proxy logs on node with explicit kubelet port", func() {
     nodeProxyTest(f, version, fmt.Sprintf(":%v/logs/", ports.KubeletPort))
})
```
Then no need for the comment and we only have one thing to keep in sync which is the pkg/master/ports constants.